files/build_templates/sonic_debian_extension.j2

```diff
@@ -579,13 +579,15 @@ sudo mkdir $FILESYSTEM_ROOT/target
 sudo mount --bind target $FILESYSTEM_ROOT/target
 sudo chroot $FILESYSTEM_ROOT docker $SONIC_NATIVE_DOCKERD_FOR_DOCKERFS info
 {% for image in installer_images.strip().split(' ') -%}
-{% set imagefilename = image.split('/')|last -%}
+{% set imagefilepath = image.split(':')|first -%}
+{% set imageversion = image.split(':')|last -%}
```

Comment:
We might need a way to control as remain in current mode (_use image_version_) or adopt new. If not this could break some.

Does this PR allow for a way to stay in current format?

Reply:
Actually I do not require it to be tagged with a version, a version is in the manifest, so if you're saying it can break someone I return the old tag

Reply:
Let us keep the current behavior, as much possible, unless we need to change.